build.gradle

```diff
@@ -70,16 +71,14 @@ ext {
     testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
 }
 
-group = ext.groupId
-
 allprojects {
     repositories {
         // Needed to resolve newest support libraries with IntelliJ / Gradle
         google()
         jcenter()
         maven {
-            // Repository for Cyface libraries
-            url 'https://dl.bintray.com/cyface-de/android-utils'
+            // Repository for Cyface Android Utils
+            url = uri("https://maven.pkg.github.com/cyface-de/android-utils")
```

Comment:
Our android libraries are now all published to the Github Package Registry to reduce the number of 3rd party apps we are using. This, I had to change the repository here as the backend has this lib as dependency.